pandas/tests/test_common.py

```diff
@@ -251,8 +263,18 @@ def test_compression_size_fh(obj, method, compression_only):
         assert uncompressed > compressed
 
 
-# GH 21227
 def test_compression_warning(compression_only):
+    """
```

Comment:
Minor nit but I'd rather see this as comments instead of a docstring, just so it doesn't clutter any test failure output

Reply:
Didn't realize docstrings get printed when the test fails... that could be nice in many cases.

Happy to change docstring to comment here. But were this test to fail, wouldn't it be helpful for this message to be outputted?

Reply:
I'm not aware of anywhere else where we do this so don't want to introduce any special cases for now. Could always be changed in the future in perhaps a separate, more comprehensive PR if there's a need for it and other test docstrings

Reply:
Changed test docstrings to comments in https://github.com/pandas-dev/pandas/pull/22011/commits/6be808d87f8470b733534969397613784eb35a1c. For the record, I think it's a bit awkward to use comments for test function descriptions, especially since it's less clear whether the comment documents the whole function or just a single part.